.github/workflows/ci-cd.yml

```
@@ -0,0 +1,168 @@
+    env:
+      ENVIRONMENT: development
+      DEBUG: true
+      LOG_LEVEL: INFO
+      GITHUB_ACTIONS: true
+      
+    steps:
+    - name: 코드 체크아웃
+      uses: actions/checkout@v4
+      
+    - name: Python 환경 설정
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)